### PR TITLE
Implemented Check ship function.

### PR DESCRIPTION
**Pull request by Pravallika2812**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -80,7 +99,14 @@ def createShip():
 Returns: bool
 '''
 def checkShip(grid, ship):
-    return
+    count=0
+    for i in ship:
+        if grid[i[0]][i[1]]==1:
```
Comment (Pragya28) [inline battleship.py:104]: Use the variable name in place of the number.

File: battleship.py [hunk 2]
```diff
@@ -80,7 +99,14 @@ def createShip():
...
 def checkShip(grid, ship):
-    return
+    count=0
+    for i in ship:
+        if grid[i[0]][i[1]]==1:
+            count+=1
+            if count==3:
+              return True
+        else:
+            return False
```
Comment (Pragya28) [inline battleship.py:109]: Reduce the number of lines in the function by checking the not equal condition and returning False, and returning True on completion of the loop.

**Review by Pragya28 — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
Comment (Pravallika2812) [reply battleship.py]: Modified entire code and changed variable name

File: battleship.py [hunk 3]
```diff
@@ -101,12 +101,11 @@ def createShip():
 def checkShip(grid, ship):
     count=0
```
Comment (Pragya28) [inline battleship.py:102]: This variable is not required. Please remove it.

File: battleship.py [hunk 4]
```diff
@@ -80,7 +99,13 @@ def createShip():
 Returns: bool
 '''
 def checkShip(grid, ship):
-    return
+    count=0
```
Comment (Pragya28) [inline battleship.py:102]: This variable is not required.

**Review by Pragya28 — CHANGES_REQUESTED** [review]

**Review by Pragya28 — COMMENTED** [review]
Good work @Pravallika2812.

**Pull request merged**